### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by brunoartc**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3756,13 +3811,19 @@ <h1 id="availability_points">
...
+
+                      
                       const idx = this.days.findIndex(d => d.id === day.id);
                       if (idx >= 0) {
                         this.removeDays.add(day.id)
                         this.days.splice(idx, 1);
                       } else { //Add days
 
                         if (curatedHouse == false) {
+                          console.log(day + curatedHouse)
```
Comment (brunoartc) [inline index.html:3824]: remover console log

File: index.html [hunk 2]
```diff
@@ -3756,13 +3811,19 @@ <h1 id="availability_points">
                   },
                   methods: {
                     onDayClick(day) {
+                      console.log("aa"+curatedHouse)
```
Comment (brunoartc) [inline index.html:3814]: remover console log

File: index.html [hunk 3]
```diff
@@ -1550,37 +1576,56 @@ <h1>
...
                     $("#cpf").inputmask("999.999.999-99");
 
 
                   } else {
+                    //Reset
+                    document.getElementById("cpf").setAttribute("minlength", "0");
+                    document.getElementById("cpf").setAttribute("maxlength", "524288");
+                    
                     // document.getElementById("docLabel").innerHTML = "Passaporte*";
+                    // document.getElementById("cpf").reset();
```
Comment (brunoartc) [inline index.html:1607]: remove codigo nao usado

File: index.html [hunk 4]
```diff
@@ -1392,10 +1410,18 @@ <h1>
             var daysObjectList = JSON.parse($("#days_list").html())
             var daysList = daysObjectList.map(a => a.id);
 
+            console.log("DIASSSS")
```
Comment (brunoartc) [inline index.html:1413]: remove console.log

**Review by andrekwr — COMMENTED** [review]
Removed console logs

**Review by andrekwr — APPROVED** [review]

**Pull request merged**